src/features/applicationView/offerCard/sendInvoiceForm/SendInvoiceForm.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import * as Yup from 'yup';
+import { useTranslation } from 'react-i18next';
+import { TextInput } from 'hds-react';
+import { Form, Formik } from 'formik';
+
+import styles from './sendInvoiceForm.module.scss';
+import FormHeader from '../../../../common/formHeader/FormHeader';
+import Button from '../../../../common/button/Button';
+import Text from '../../../../common/text/Text';
+
+type FormValues = {
+  dueDate: Date;
+};
+
+export type SendInvoiceFormProps = {
+  email: string | null;
+  isSubmitting: boolean;
+  onCancel: () => void;
+  onSubmit: (data: FormValues) => void;
+};
+
+const SendInvoiceForm = ({ email, onSubmit, onCancel, isSubmitting }: SendInvoiceFormProps) => {
+  const { t } = useTranslation();
+
+  const validationSchema = Yup.object<FormValues>().shape({
```

Comment:
Consider moving validationSchema outside of the component. Now there is unnecessary Yup.object function call on every render.

Reply:
Good catch, fixed

Reply:
Added another comment, there is still function call on every render :)